components/board/jetson/data.go

```diff
@@ -17,198 +17,203 @@ const (
+	{Name: "19", DeviceName: "UNKNOWN169", LineNumber: 162, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "21", DeviceName: "UNKNOWN169", LineNumber: 161, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "22", DeviceName: "UNKNOWN169", LineNumber: 150, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "23", DeviceName: "UNKNOWN169", LineNumber: 160, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "24", DeviceName: "UNKNOWN169", LineNumber: 163, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "26", DeviceName: "UNKNOWN169", LineNumber: 164, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "29", DeviceName: "UNKNOWN169", LineNumber: 105, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "31", DeviceName: "UNKNOWN169", LineNumber: 106, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "32", DeviceName: "UNKNOWN169", LineNumber: 108, PwmChipSysfsDir: "32f0000.pwm", PwmID: 0},
+	{Name: "33", DeviceName: "UNKNOWN169", LineNumber: 84, PwmChipSysfsDir: "3280000.pwm", PwmID: 0},
+	{Name: "35", DeviceName: "UNKNOWN169", LineNumber: 130, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "36", DeviceName: "UNKNOWN169", LineNumber: 113, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "37", DeviceName: "UNKNOWN169", LineNumber: 151, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "38", DeviceName: "UNKNOWN169", LineNumber: 129, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "40", DeviceName: "UNKNOWN169", LineNumber: 128, PwmChipSysfsDir: "", PwmID: -1},
 }
 
+// It's not clear that this has ever worked properly: back when we identified chips by the ngpio
+// number, the two chips supposedly had 169 and 30 lines on them, but folks online seem to claim
+// that the Xavier's chips have 223 and 39 lines. So, these are probably broken right now but could
+// be fixed with some inspection of an actual Xavier board (which we don't have in the office).
+// More likely, they'll be fixed when someone creates a customlinux pin definition file.
 var jetsonXavierPins = []genericlinux.PinDefinition{
-	{Name: "7", Ngpio: 169, LineNumber: 106, PwmChipSysfsDir: "", PwmID: -1},
-	{Name: "11", Ngpio: 169, LineNumber: 112, PwmChipSysfsDir: "", PwmID: -1},
-	{Name: "12", Ngpio: 169, LineNumber: 51, PwmChipSysfsDir: "", PwmID: -1},
-	{Name: "13", Ngpio: 169, LineNumber: 108, PwmChipSysfsDir: "32f0000.pwm", PwmID: 0},
+	{Name: "7", DeviceName: "UNKNOWN169", LineNumber: 106, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "11", DeviceName: "UNKNOWN169", LineNumber: 112, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "12", DeviceName: "UNKNOWN169", LineNumber: 51, PwmChipSysfsDir: "", PwmID: -1},
+	{Name: "13", DeviceName: "UNKNOWN169", LineNumber: 108, PwmChipSysfsDir: "32f0000.pwm", PwmID: 0},
 	// Older versions of L4T don't enable this PWM controller in DT, so this PWM
 	// channel may not be available.
-	{Name: "15", Ngpio: 169, LineNumber: 84, PwmChipSysfsDir: "3280000.pwm", PwmID: 0},
-	{Name: "16", Ngpio: 30, LineNumber: 8, PwmChipSysfsDir: "", PwmID: -1},
-	{Name: "18", Ngpio: 169, LineNumber: 44, PwmChipSysfsDir: "32c0000.pwm", PwmID: 0},
```

Comment:
It's not obvious to me that this has ever worked. These were looking for chips with 169 and 30 lines on them, but the NVidia forms imply that the Jetson Xavier's chips should have 224 and 40 pins. So, I set the device names to "UNKNOWN" with the old ngpio number, and we can hopefully change these to correct values when we (or a user who wants to help us out) figure out what's really going on.